CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # CHANGELOG
 
-## Next Release
+## On Demo
 ## Internal
 - Remove old AllPatientsInFacilityView code
 - Make changing of screens in Flow synchronous
 
-## On Demo
+## Pending Release
```

Comment:
Shouldn't this be the release name?

Reply:
The release hasn't gone out yet. We still don't know what artifact will go out tomorrow. Hence.

Reply:
This is done.